tools/pot/openvino/tools/pot/algorithms/quantization/fast_bias_correction/algorithm.py

```diff
@@ -215,4 +215,6 @@ def get_inputs_shape(self, nodes_with_bias):
         _, inputs_shape = self._engine.predict(calculate_input_shape, sampler)
         for node_name, shape_node in inputs_shape.items():
             inputs_shape[node_name] = shape_node['shape_node'][0]
+            if len(inputs_shape[node_name]) > 1:
```

Comment:
I wonder if we can add a test for that?

Do we have the same problem in "honest" BC?